internal/bminventory/inventory.go

```diff
@@ -1719,6 +1716,11 @@ func (b *bareMetalInventory) InstallClusterInternal(ctx context.Context, params
 		return nil, common.NewApiError(http.StatusInternalServerError, errors.New("Failed to generated additional cluster manifest"))
 	}
 
+	if err = b.providerRegistry.GenerateProviderManifests(ctx, cluster); err != nil {
```

Comment:
shouldn't we keep this call in GenerateAdditionalManifests?

Reply:
I moved this to the cluster.GenerateAdditionalManifests method